lms/tasks/email_digests.py

```diff
@@ -60,13 +62,19 @@ def send_instructor_email_digest_tasks(batch_size):
 
             for batch in batches:
                 send_instructor_email_digests.apply_async(
-                    [batch],
-                    {"updated_after": updated_after, "updated_before": updated_before},
+                    (),
+                    {
+                        "h_userids": batch,
+                        "updated_after": updated_after,
+                        "updated_before": updated_before,
+                    },
                 )
 
 
-@app.task
+@app.task(bind=True, max_retries=2)
 def send_instructor_email_digests(
+    self,
+    *,
```

Comment:
I've changed `send_instructor_email_digests()`'s arguments to be keyword-only arguments. See below for why